pkg/config/config_template.yaml

```diff
@@ -1483,6 +1483,17 @@ api_key:
   #
   # trace_buffer: 0
 
+  ## @param receiver_timeout - integer - optional - default: 5
+  ## @env DD_APM_RECEIVER_TIMEOUT - integer - optional - default: 5
+  ##
+  ## WARNING: This config should generally not be changed unless directed to by Datadog Support for debugging purposes.
+  ## Issues that may present as timeouts are frequently due to resource constraints and increasing this value is unlikely
+  ## to help.
+  ##
+  ## Specifies the timeout in number of seconds for trace-agent endpoints.
```

Comment:
We should move this before the warning (first we explain what it does then we add the warning).